Dx 387 atualizar documentacao para incluir status de cada

ArturGuedes opened the pull request.

riessnico commented on compliance/compliance-requirements.mdx:
```diff
@@ -88,4 +92,16 @@ If the entity's provided information is incomplete or inconsistent, Lumx may req
\ No newline at end of file
+In some cases, an entity may be rejected if the submitted information is invalid or inconsistent, such as an incorrect tax identification number or unrecognized documents.
+
+### Checking Beneficiary Status
```
Cara acho que não necessário isso aqui. Melhor tirar ou mover para a página de beneficiaries

riessnico reviewed (changes requested):
Bom trabalho!
Ficou faltando mexer na api reference para mostrar como é o retorno do read a customer verification quando tem beneficiários.

ArturGuedes committed:
docs(api): add examples and schemas for customer verification responses

Add response examples for individual and business customer verification endpoints
Include detailed schemas for verification status, documents, and beneficiaries

ArturGuedes committed:
refactor(api): remove unused rejectSubLabels from schema

Clean up the OpenAPI schema by removing the unused rejectSubLabels field which is no longer needed in the customer verification component.

ArturGuedes committed:
docs(compliance): remove beneficiary status section from requirements

The details of the beneficiary status verification have been removed
